web_warning_on_save/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Raise warning when saving
+=========================
+
+This module was written to extend the functionality of saving a record in the web interface.
+/!\/!\/!\ In no way this module stops the save of the record. You must consider this as a warning displayed 
+to the user AFTER save completed. /!\/!\/!\
+If you don't want OpenERP to save the record, you should use constraints.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* write a method called 'check_warning_on_save' which will make some checks and return a string
+
+example :
```

Comment:
You might need to write

`example::`

In order to tell following text is code

or maybe even

    example :

    .. code:: python

However, I'm not sure it will display well on Odoo